tests/integration/synapseclient/models/async/test_submission_async.py

```diff
@@ -1,199 +1,674 @@
-def test_create_submission_async():
```

Comment:
Is this file testing the non-OOP code? I'm assuming that code is deprecated and hasn't been removed yet. If so, should these tests remain until it is sunsetted?